api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -191,6 +191,10 @@ message JwtProvider {
   //       exp: 1501281058
   //
   string payload_in_metadata = 9;
+
+  // Specify the clock skew in seconds when verifying JWT time constraint,
+  // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
+  uint32 clock_skew = 10;
```

Comment:
Why not a protobuf Duration?

Reply:
All JWT time constraints are using unit `seconds`, e.g. `esp`, `nbf`.   This is to be consist with that.  protobuf Duration allow smaller than `seconds`, which is not supported, and could be mis-leading.

Reply:
Please name this `clock_skew_seconds` then to be precise.

Reply:
Done